contrib/babelfishpg_common/src/varchar.c

```diff
@@ -92,7 +92,7 @@ is_basetype_nchar_nvarchar(Oid typid)
  * GetUTF8CodePoint - extract the next Unicode code point from 1..4
  *					  bytes at 'in' in UTF-8 encoding.
  */
-static inline int32_t
+inline int32_t
```

Comment:
don't need inline here .

Reply:
Ack. made the changes in the next revision.